### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by phongnhhn92** [aim-uofa/AdelaiDepth#17]
Hi, 
I think your model is great. Can u provide the code for the 3 training losses(Pair-wise normal loss, Image-level normalized regression loss and the multi-scale gradient loss) you have used in your recent paper ?

**Comment by YvanYin** [comment]
Hi as there are some IP issues, I cannot release them all now. For the multi-scale gradient loss, you can check here:https://github.com/aim-uofa/AdelaiDepth/issues/15.   For the Image-level normalized regression loss, you can follow the paper to implement it. It is very straight forward. You can also post your code here, then I can add comments.

**Comment by guangkaixu** [comment]
@phongnhhn92 Hi, the training code of LeReS is released today.
